### ISSUE TITLE
Character stuck between life and death

### ISSUE THREAD

**Issue by Quretaz** [dalaranwow/dalaran-wow#5594]
Hello, 
I got a char on the pvp server of Dalaran, a mage named Azuru.
Its stuck between life and death, with 3 mobs beating my body on the ground while it says im "dead".
I do not spawn as a ghost at the graveyard, im just stuck, laying on the ground in a cave.
Each time i login to the server, and want to play on this char, its the same.

Can anyone solve this problem so i can play on my mage again? 

[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Quretaz** [comment]
Here is a screenshot.

https://gyazo.com/5827e5b81f4cb14cce47502218be01b3

**Comment by Quretaz** [comment]
Sorry for the inconvenience, it seems like the problem has been fixed. A moderator can close/delete this thread.
